config/channel/consolidated/deployments/webhook.yaml

```diff
@@ -66,7 +66,9 @@ spec:
             httpHeaders:
             - name: k-kubelet-probe
               value: "webhook"
-        livenessProbe: *probe
+        livenessProbe:
+          <<: *probe
+          initialDelaySeconds: 20
```

Comment:
@matzew we need the same in KafkaSource webhook too:

https://github.com/knative-sandbox/eventing-kafka/blob/3b547e91c5511bdda7a0f82aa2cabf713894c023/config/source/deployments/controller.yaml#L63

IMO we should backport this to 0.18 for sure. I also prefer this being backported to 0.17.

Other than that, I verified this fix locally. Fixes the problem.

Reply:
@aliok +1 on patch 0.17 and 0.18